cetmix_tower_server/static/description/index.html

```diff
@@ -469,7 +469,8 @@ <h2><a class="toc-backref" href="#toc-entry-10">16.0.1.0.10 (2025-05-24)</a></h2
 <div class="section" id="section-8">
 <h2><a class="toc-backref" href="#toc-entry-11">16.0.1.0.9 (2025-05-23)</a></h2>
 <ul class="simple">
-<li>Bugfixes: Error when rendering a file not attached to a server. (4715)</li>
```

Comment:
Same here. Looks like the pre-commit is changing the files it should not change.